Jenkinsfile

```diff
@@ -210,15 +210,15 @@ sed -e 's#{BUILD_BRANCH}#${env.BRANCH_NAME}#g' \
              mkdir -p zowe-build/${env.BRANCH_NAME}_${env.BUILD_NUMBER} &&
              cd zowe-build/${env.BRANCH_NAME}_${env.BUILD_NUMBER} &&
              git clone --branch master https://github.com/zowe/zowe-dockerfiles.git
-             cd zowe-dockerfiles/dockerfiles/zowe-release/s390x/server-bundle &&
+             cd zowe-dockerfiles/dockerfiles/zowe-release/s390x/zowe-v1-lts &&
```

Comment:
should this be `server-bundle` instead? @1000TurquoisePogs

Reply:
No because it's the folder structure here https://github.com/zowe/zowe-dockerfiles/tree/master/dockerfiles/zowe-release/s390x
The name of the image and the name of the directory have diverged.